dfs/volume.go

```diff
@@ -37,11 +37,15 @@ func (dfs *DistributedFilesystem) GetVolume(serviceID string) (*volume.Volume, e
 }
 
 // GetSubvolume gets the path of the *local* volume on the host
-func GetSubvolume(fsType, varpath, serviceID string) (*volume.Volume, error) {
+func GetSubvolume(fsType, varpath, serviceID string, mountVolume VolumeMounter) (volume.Volume, error) {
```

Comment:
Hmm. I'm not sure how that would work from a unit-testing perspective. 
For example, see `TestSnapshot_Snapshot_VolumeNotFound()` in dfs/snapshot_test.go. I needed a way to return a mock instance of Volume so I that could simulate different kinds of error conditions in the unit-test.  To do that, I injected a the VolumeMounter function in the tests which returns a mock of volume.Volume.